metpy/calc/basic.py

```diff
@@ -88,6 +92,14 @@ def wind_direction(u, v):
     origshape = wdir.shape
     wdir = atleast_1d(wdir)
     wdir[wdir <= 0] += 360. * units.deg
+
+    # Handle oceanographic convection
+    if convention == 'to':
+        wdir -= 180 * units.deg
+        wdir[wdir <= 0] += 360. * units.deg
```

Comment:
Instead of repeating this line here, could we simply remove it above do it once below everything else, before we return?

Reply:
Yup, I said to myself that I shouldn't be repeating this line when I wrote it. Again, more coffee.

Reply:
Progress! 😉